examples/mnist/assets/objective/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# this base image works in both CPU and GPU enabled environments
+FROM substrafoundation/substra-tools:0.5.0
+
+# install dependencies
+RUN pip3 install sklearn==0.0
```

Comment:
0.0 ?

Reply:
il faut copier le requirements.txt et `pip3 install -r requirements.txt`

Reply:
Oh I got the same _version_: 
```sh
pip3 list | grep sklearn
sklearn  0.0
```